scripts/extension-data.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * This file generates pkg/rancher-desktop/assets/extension-data.yaml
+ *
+ * Usage: `yarn generate:extension-data`
+ */
+
+import childProcess from 'child_process';
+import fs from 'fs';
+import util from 'util';
+
+import yaml from 'yaml';
+
+/**
+ * This is a fixed list of extension image names, mapping to whether this
+ * extension is compatible with containerd.  Only these extensions will be
+ * scanned.
+ */
+const extensionInfo = {
```

Comment:
I guess this could be refactored in a later PR, but it would be good to store the extension names and versions in way similar to `dependencies.yaml`, so that this could one day be integrated into `rddepman` to check if any bundled extension has updates available.

Reply:
I did it in this PR instead, _solely_ so I can get the spell checker to ignore then entire file and be done with it.